README.md

```diff
@@ -1,4 +1,32 @@
-geo-location
-================
+[![Published on webcomponents.org](https://img.shields.io/badge/webcomponents.org-published-blue.svg)](https://www.webcomponents.org/element/ebidel/geo-location)
 
-See the [documentation](https://ebidel.github.io/geo-location/) page.
+## \<geo-location\>
+
+Geolocation API Polymer web component.
+
+Example to get the device geolocation values:
+<!---
+```
+<custom-element-demo>
+  <template>
+    <script src="../webcomponentsjs/webcomponents-lite.js"></script>
+    <link rel="import" href="geo-location.html">
+    <div>
+      <dom-bind>
+        <template is="dom-bind">
```

Comment:
@ebidel Please note, this is hybrid, not Polymer 2.x-only code. See https://www.polymer-project.org/2.0/docs/upgrade#convert-template-extension-elements-at-the-document-level. `<dom-bind>` is Polymer 2.x-only element. If I understand correctly, in this case `<template is="dom-bind">`

1. at the document level
2. **not nested** extension element

so it must be wrapped manually.